README.md

```diff
@@ -0,0 +1,75 @@
+# docker-logs-collector
+
+docker-logs-collector is a Docker container that uses Fluent Bit to collect logs from other Docker containers and forward those logs to your Logz.io account.
+
+To use this container, you'll set environment variables in your `docker run` command.
+docker-logs-collector uses those environment variables to generate a valid Fluent Bit configuration for the container.
+docker-logs-collector mounts docker.sock and the Docker logs directory to the container itself, allowing Fluent Bit to collect the logs and metadata.
+
+docker-logs-collector ships logs only.
+If you want to ship metrics to Logz.io, see [docker-collector-metrics](https://github.com/logzio/docker-collector-metrics).
+
+**Note:**
+- Ensure your Fluent Bit configuration matches your logging requirements and environment variables are set correctly.
+
+## docker-logs-collector setup
+
+### 1. Pull the Docker image
+
+Download the appropriate Docker image for your architecture (amd64 or arm64):
+
+```shell
+# For amd64 architecture
+docker pull logzio/docker-logs-collector-amd64:latest
+
+# For arm64 architecture
+docker pull logzio/docker-logs-collector-arm64:latest
+```
+
+### 2. Run the container
+
+Replace `<ARCH>` in the image name with either amd64 or arm64 based on your system architecture.
+
+For a complete list of options, see the parameters below the code block.👇
+
+```shell
+docker run --name docker-logs-collector \
+--env LOGZIO_LOGS_TOKEN="<LOGS-SHIPPING-TOKEN>" \
+-v /var/run/docker.sock:/var/run/docker.sock:ro \
+-v /var/lib/docker/containers:/var/lib/docker/containers \
+-e HEADERS="user-agent:logzio-docker-logs" \
+logzio/docker-logs-collector-<ARCH>:latest
+```
```

Comment:
I chose this approach because separating images per architecture simplify testing, debugging, and deployment, offering precise control over each architecture without the complexity of a multi-arch build.

Reply:
I took you suggestion and I streamlined the Dockerfile for both amd64 and arm64 to support multi-architecture builds.